### PR TITLE
team join: you've already requested to join that team

### PR DESCRIPTION
**Pull request by idrysdale**
This should be fine, now that https://github.com/fluidkeys/fluidkeys/pull/411 is in.

### CODE REVIEW

File: fk/teamjoin.go [hunk 1]
```diff
@@ -69,6 +73,26 @@ func teamJoin(teamUUID uuid.UUID) exitCode {
 
 }
 
+func ensureNoExistingRequests(teamUUID uuid.UUID, fingerprint fpr.Fingerprint) exitCode {
+	existingRequest, err := db.GetExistingRequestToJoinTeam(teamUUID, fingerprint)
+	if err != nil {
```
Comment (paulfurley) [inline fk/teamjoin.go]: this catches `ErrRequestNotFound` and therefore prevents you getting to line 82:

image

Comment (idrysdale) [reply fk/teamjoin.go]: Good spot!

Do you think having this `ErrRequestNotFound` on the query is correct?

Perhaps we should just return no error, but nil request?

Comment (paulfurley) [reply fk/teamjoin.go]: `nil` request with no error works for me

Comment (idrysdale) [reply fk/teamjoin.go]: 👍

Comment (idrysdale) [reply fk/teamjoin.go]: Because of this change https://github.com/fluidkeys/fluidkeys/pull/411 this error won't raise on no errors now.

**Pull request merged**